src/modules/Providers/EmptyStateProviders.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import RedHatProgressionIcon from '_/components/RedHatProgressionIcon';
+import { useTranslation } from '_/utils/i18n';
+
+import * as APP_C from '@app/common/constants';
+import {
+  Bullseye,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Text,
+  TextContent,
+  TextList,
+  TextListItem,
+  Title,
+} from '@patternfly/react-core';
+
+import { AddProviderButton } from './ProvidersPage';
+
+const EmptyStateProviders: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <EmptyState variant={EmptyStateVariant.large} isFullHeight>
+      <EmptyStateIcon icon={RedHatProgressionIcon} />
```

Comment:
https://www.patternfly.org/v4/components/empty-state/

icon can be `React.ComponentType<any>`
you can actully do `icon={() => <img src={some image} />}`
then we can keep the image in an images directory and it may be easier to handle them, mainly update the image when we find a nicer more colorful image we like

Reply:
We'd need to use something other than empty state for a different layout anyway.  The PF design guidelines are useful here:
   https://www.patternfly.org/v4/components/empty-state/design-guidelines

I'd rather just keep the svg icon and replace with a new image icon in future when the layout changes.